LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java

```diff
@@ -7,6 +7,10 @@
 import androidx.test.core.app.ApplicationProvider;
```

Comment:
This file and other test files should also change their package structure, correct?